modules/renderer/photos.js

```diff
@@ -192,11 +192,11 @@ export function rendererPhotos(context) {
             // support opening a photo via a URL parameter, e.g. `photo=mapillary-fztgSDtLpa08ohPZFZjeRQ`
```

Comment:
I think we should add examples of all the photo services up here so we can easily see which cases we need to check for